app/models/bid.rb

```diff
@@ -1,2 +1,4 @@
 class Bid < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Good question.  That will be an interesting validation.  We can prevent the bid button from appearing on the screen for them, but we should prob back that up with a validation as well.  You want me to work on that?